quickstarts/microsoft.documentdb/cosmosdb-sql-rbac/main.bicep

```diff
@@ -0,0 +1,68 @@
+@description('Location for all resources.')
+param location string = resourceGroup().location
+
+@description('Cosmos DB account name, max length 44 characters')
+param accountName string = 'sql-rbac-${uniqueString(resourceGroup().id)}'
+
+@description('Friendly name for the SQL Role Definition')
+param roleDefinitionName string = 'My Read Write Role'
+
+@description('Data actions permitted by the Role Definition')
+param dataActions array = [
+  'Microsoft.DocumentDB/databaseAccounts/readMetadata'
+  'Microsoft.DocumentDB/databaseAccounts/sqlDatabases/containers/items/*'
+]
+
+@description('Object ID of the AAD identity. Must be a GUID.')
+param principalId string
+
+var account_name = toLower(accountName)
```

Comment:
is this necessary, seems like we could just tolower on the property assignment?

Reply:
updated.